loop/storage/redis.js

```diff
@@ -940,6 +957,7 @@ RedisStorage.prototype = {
       'userRooms.' + userMac,
       'room.' + roomToken
     );
+    multi.expire('userRooms.' + userMac, self._settings.hawkSessionDuration);
```

Comment:
This is the owner room list. If the owner doesn't exist anymore it's room list doesn't.